var/spack/repos/builtin/packages/w3emc/package.py

```diff
@@ -56,7 +57,7 @@ class W3emc(CMakePackage):
 
     def setup_run_environment(self, env):
         if self.spec.satisfies("@:2.9"):
-            suffixes = ["4", "d", "8"]
+            suffixes = ("4", "d", "8")
```

Comment:
@AlexanderRichert-NOAA should these be parens or square brackets?

Reply:
Doesn't technically matter, but some Spack devs have been pushing for using tuples (the parens) when possible, so I'd say let's use parens for consistency.